fix "LNK1118: syntax error in 'VERSION' statement" (MSVC only accepts a major.minor)

--- a/zlib/contrib/vstudio/vc17/zlibvc.def	2024-01-22 10:32:38.000000000 +0100
+++ b/zlib/contrib/vstudio/vc17/zlibvc.def	2026-01-09 14:52:37.858518600 +0100
@@ -1,7 +1,7 @@
 LIBRARY
 ; zlib data compression and ZIP file I/O library
 
-VERSION		1.3.1
+VERSION		1.3
 
 EXPORTS
         adler32                                  @1
